AirCasting/CoreData/MeasurementStream.swift

```diff
@@ -0,0 +1,19 @@
+// Created by Lunar on 07/05/2021.
+//
+
+import Foundation
+
+struct MeasurementStream: Hashable {
+    let id: MeasurementStreamID?
```

Comment:
You answered the first part during the call but I'm leaving the comment for the  implicitly unwrapped optional suggestion.

Reply:
When the MeasurementStream is not synced with server it will not have an id. That why MeasurementStreamLocalID was created so we can reference it before sync